authorizer/src/main/res/values-nb-rNO/strings.xml

```diff
@@ -0,0 +1,690 @@
+    <string name="passwords_do_not_match">Passordene samsvarer ikke</string>
+    <string name="permissions">Tilganger</string>
+    <string name="policy_header">Praksis</string>
+    <string name="policy_no">Nei</string>
+    <string name="policy_yes_len">Ja — Ihvertfall %1$d</string>
+    <string name="policy_yes_sym_default" tools:ignore="PluralsCandidate">Ja — Ihvertfall %1$d fra forvalgte symboler: %2$s</string>
+    <string name="policy_yes_sym_easy" tools:ignore="PluralsCandidate">Ja — Ihvertfall %1$d fra enkle symboler: %2$s</string>
+    <string name="policy_yes_sym_policy" tools:ignore="PluralsCandidate">Ja — Ihvertfall %1$d fra praksissymboler: %2$s</string>
+    <string name="policy_yes_sym_pronounce" tools:ignore="StringFormatCount">Ja — Symboler: %2$s</string>
+    <string name="press_again_close_warning">Trykk igjen for å lukke filen</string>
+    <string name="press_password_warning">Trykk «Passord» igjen for å lime inn i synlig felt</string>
+    <string name="press_yubikey">Trykk «YubiKey»</string>
+    <string name="protect_all">Beskytt alle oppføringer</string>
+    <string name="protect_group">Beskytt gruppeoppføringer</string>
+    <string name="protected_record">Beskyttet</string>
+    <string name="protected_record_header">Beskyttet oppføring</string>
+    <string name="psafe3_ext">.psafe3</string>
+    <string name="query_label">Søk: %1$s</string>
+    <string name="recent_files">Nylige filer</string>
+    <string name="record">Oppføring</string>
+    <string name="records">Oppføringer</string>
+    <string name="record_has_references">Oppføringen har referanser</string>
+    <string name="record_not_found">Fant ikke oppføring</string>
+    <string name="record_policy">Oppføringspraksis</string>
+    <string name="record_sort_order">Sorteringsrekkefølge for oppføringer</string>
+    <string name="record_type">Oppføringstype</string>
+    <string name="read_only_about">Skrivebeskyttet</string>
+    <string name="read_only_file">Skrivebeskyttet</string>
+    <string name="read_only_media">Filen finnes på skrivebeskyttet lagring. Situasjonen kan oppstå på Kitkat og nyere med fjernbare SD-kort.</string>
+    <string name="read_write">Les og skriv</string>
+    <string name="references_label">Referanser</string>
+    <string name="regular_expression">Regulært uttrykk</string>
+    <string name="regular_expression_summary">Bruk et regulært uttrykk for å søke istedenfor en understreng</string>
+    <string name="remove">Fjern</string>
+    <string name="recurring">Gjentagende</string>
+    <string name="save">Lagre</string>
+    <string name="save_password">Lagre passord</string>
+    <string name="save_password_p">Lagre passord?</string>
+    <string name="save_password_warning">
+        Etter at filen er åpnet, vil innskrevet passord lagres lokalt.
+        Passordet er kryptert med en AES-256-nøkkel og er beskyttet av
+        en fingeravtrykksskanning. Fingeravtrykksskanningen er ikke nødvendigvis
```

Comment:
Unclear what "may" meant here.
https://datatracker.ietf.org/doc/html/rfc2119#section-5
Used ~"Not necessarily as safe as the password itself".